source/install-desktops.rst

```diff
@@ -0,0 +1,22 @@
+.. _install-desktops:
+
+Install Desktops
+================
+
+This installation guide will walk you through setting up an interactive Desktop
+App that your users will be able to use to launch an interactive Desktop
+session within a batch job on a compute node. The user should then be able to
+connect to a running session through their browser using the `noVNC`_ client.
```

Comment:
I wonder if we should introduce the concept of "interactive HPC" here... Instead of "Install Desktops" should it be called "Install Interactive HPC Desktops"